HTTP_HEADER_FORMAT.md

```diff
@@ -61,26 +60,97 @@ caller rather than strict rules to follow for 3 reasons:
 3.  Different load between caller service and callee service might force callee
     to down sample.
 
-The behavior of other bits is currently undefined.
+### Prefix '.' The Hierarchical-id
```

Comment:
This is minor issue. What is unique about the Hierarchical-id is that it is a vector of logical clocks, i.e. unlike the other identifiers, its values have a sort. So consider renaming this to be just Vector or something similar. Yes, Hierarchical-Ids also work, but the structure generated by Span-Ids is also hierarchical, and does not quite reflect its uniqueness.

Reply:
I am OK with that.   We should do as a follow-on PR.